07_future_promise/homework/task1/src/main.cpp

```diff
@@ -14,7 +17,34 @@ int main()
 		"dev4@company.com",
 		"dev5@company.com",
 	};
-	std::string message = "build failed";
+	
+	std::vector<std::future<bool>> queue;
+
+	auto start = std::chrono::high_resolution_clock::now();
+	for (auto &&i : responsible_devs) {
+		queue.push_back(
+			std::async(std::launch::async, [&i]
+				{
+					std::stringstream ss;
+					bool result  = libmail::send_mail(i, "message");
+					std::string r = "FAIL";
+					if(result) { r="OK"; } ;
+
+					ss << "Sending to " << i << " " << r;
+					ss << std::endl;
```

Comment:
Tutaj nie powinno być wypisywania. To powinien główny wątek robić na podstawie wyników, które async zwracają.